contracts/cw-governance/src/contract.rs

```diff
@@ -59,7 +71,48 @@ pub fn instantiate(
 
     GOVERNANCE_MODULE_COUNT.save(deps.storage, &(gov_module_msgs.len() as u64))?;
 
-    Ok(Response::new()
+    let mut response = Response::new();
```

Comment:
It may be more readable to make a `Vec<Submsg<Empty>>` here instead of adding these to the response directly. You can then call `add_submessages ` later like we do with the `gov_module_msgs`.